hook-examples/fetchproductiondb/fetchproductiondb

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+## Description: Open PHPStorm with the current project
+## Usage: fetchproductiondb
+## Example: "ddev fetchprodctiondb"
+
+# Example is macOS-specific, but easy to adapt to any OS
+ssh -t user@host 'mysqldump --defaults-file=~/.mysql/project.cnf databasename' > productionsqldump.sql
+ddev import-db --src=productionsqldump.sql
```

Comment:
FYI `ddev import-db` (and also `ddev mysql`) are happy to accept from stdin, so it's not actually necessary to dump to a file first. (Or end up having to delete the file)

Reply:
mhm ok, we decided to have the step as we can then easier debug the output by simply adding a pause or commenting the rm - but i will think about that

Reply:
the web command sounds like a good plan - will check it out